src/components/BaseLayerToggler/BaseLayerToggler.js

```diff
@@ -236,10 +236,11 @@ class BaseLayerToggler extends Component {
     } else {
       newIdx = idx + 1;
     }
-
+    const newLayerVisible = layers[idx === -1 ? 0 : idx];
     this.setState({
       layers,
       idx: layers.length > 1 ? newIdx : null,
+      layerVisible: newLayerVisible,
```

Comment:
Would this work as well? That would be the same behavior as in `updateState()`.
```suggestion
      layerVisible: layers.length > 1 ? layers[newIdx] : null,
```